userguide/system.rst

```diff
@@ -2136,53 +2136,55 @@ system, then run the update program, giving it the path to the file:
 Updating an HA System
 ~~~~~~~~~~~~~~~~~~~~~
 
-If the %brand% array has been configured for High Availability
-(HA), the update process must be started on the active |ctrlr-term|.
-
-%brand% downloads the update files to both |ctrlrs-term|, then updates
-and reboots the standby |ctrlr-term|. The %brand% user manually initiates
-a :ref:`Failover` to activate the standby |ctrlr-term|. %brand%
-completes the update process by installing the update on the previously
-active |ctrlr-term|.
-
-A dialog describing this process is shown when an HA update is started.
-Clicking :guilabel:`CONTINUE` starts the update.
-
-.. note:: Other users logged in to the |web-ui| are notified that an
-   update is in progress and that the system will restart when the
-   update is complete.
+On the
+:menuselection:`Dashboard`
+of the |ctrlr-term-active|, click :guilabel:`CHECK FOR UPDATES`. This
+button changes to :guilabel:`UPDATES AVAILABLE` when there is an
+available update. Clicking the button goes to
+:menuselection:`System --> Update`.
+Clicking :guilabel:`DOWNLOAD UPDATES` opens a dialog to save the
```

Comment:
We are still implying to the user that a button labeled "DOWNLOAD UPDATES" actually does not download updates.
```suggestion
When :guilabel:`DOWNLOAD UPDATES` is clicked, it first gives an
opportunity to save the current system configuration, then available
updates are downloaded.
```